docs/resources/webhook.md

```diff
@@ -13,17 +13,52 @@ Only available for dedicated subscription plans.
 
 ## Example Usage
 
+<details>
+ <summary>
+    <b>
+      <i>Enable webhook before v1.30.0</i>
+    </b>
+  </summary>
+
+Doesn't support updating the resource, makes all argument using `ForceNew` behaviour.
+Any changes to an argument will destroy and re-create the resource. Also no longer support
+for using `retry_interval` argument in the backend, even though it's set to be required.
+
 ```hcl
 resource "cloudamqp_webhook" "webhook_queue" {
   instance_id = cloudamqp_instance.instance.id
-  vhost = "myvhost"
+  vhost = cloudamqp_instance.instance.vhost
   queue = "webhook-queue"
   webhook_uri = "https://example.com/webhook?key=secret"
   retry_interval = 5
   concurrency = 5
 }
 ```
 
+</details>
+
+<details>
+ <summary>
+    <b>
+      <i>Enable webhook after [v1.30.0](https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.30.0)</i>
```

Comment:
```suggestion
      <i>Enable webhook after (v1.30.0)[https://github.com/cloudamqp/terraform-provider-cloudamqp/releases/tag/v1.30.0]</i>
```

found by looking at the rendered version at https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/webhook-retries/docs/resources/webhook.md

Reply:
Looks rather how Github interpret markdown different from https://registry.terraform.io/tools/doc-preview when used with html tags (noticed in other cases too). Both support the common syntax [text](link) in free text. 

When using link between html tags:
- doc-preview wants [text](link) between <i> works, (text)[link] doesn't.
- Github-preview get the same result between [text](link), (text)[link]. non of them render correct.

Another example:
https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/main/docs/resources/alarm.md
https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/alarm

Perhaps just remove the link or make the reference somewhere else.

Reply:
Oh 

You can make the link with HTML then, does it work for both? GitHub should allow it at least

Reply:
Works for both.